modules/acquia_cms_headless/modules/acquia_cms_headless_ui/tests/src/Functional/HeadlessFrontpageTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless_ui\Functional;
+
+use Acquia\DrupalEnvironmentDetector\AcquiaDrupalEnvironmentDetector;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests for acquia_cms_headless frontpage.
+ *
+ * @group acquia_cms_headless
+ * @group low_risk
+ */
+class HeadlessFrontpageTest extends BrowserTestBase {
```

Comment:
@deepakmishra2 can we extend DashboardBrowserTestBase instead of BrowserTestBase, if yes then it would save the code repeatition.

cc: @rajeshreeputra @vishalkhode1

Reply:
@apathak18 DashboardBrowserTestBase uses DashboardSectionTrait i dont think we need that trait, also i need UI module to test this feature whereas in DashboardBrowserTestBase it uses entire headless module.

cc @vishalkhode1